Library/Homebrew/download_strategy.rb

```diff
@@ -482,10 +482,13 @@ def fetch(timeout: nil)
         # The cached location is no longer fresh if either:
         # - Last-Modified value is newer than the file's timestamp
         # - Content-Length value is different than the file's size
-        if cached_location_valid
-          newer_last_modified = last_modified && last_modified > cached_location.mtime
-          different_file_size = file_size&.nonzero? && file_size != cached_location.size
-          cached_location_valid = !(newer_last_modified || different_file_size)
+        if cached_location_valid && last_modified && last_modified > cached_location.mtime
+          ohai "Ignoring #{cached_location}", "mtime #{cached_location.mtime.iso8601} < Last-Modified: #{last_modified&.iso8601}"
```

Comment:
Can you use `odebug` here instead?

Reply:
Sure!

Before I do, though, I feel like it should be at the same log level as [this](https://github.com/Homebrew/brew/blob/2c89b70f2863a1581c7aaf7e5d0b7ab59ed18cb5/Library/Homebrew/download_strategy.rb#L492)
```ruby
puts "Already downloaded: #{cached_location}"
```

Do you think I should switch my PR to `puts`?

Or switch them both to `odebug ... always_display: true`? or just `odebug`?